flink-python/pyflink/datastream/connectors/base.py

```diff
@@ -52,6 +53,24 @@ def __init__(self, sink: Union[str, JavaObject]):
         super(Sink, self).__init__(sink)
 
 
+class TransformAppender(ABC):
+
+    @abstractmethod
+    def apply(self, ds):
+        pass
+
+
+class PreTransformWrapper(ABC):
+
+    @abstractmethod
+    def need_pre_transform(self) -> bool:
```

Comment:
remove this method?

Reply:
This is still needed that a sink implementing `SupportsPreprocessing` may not require preprocessing, e.g. `KafkaSink` doesn't need preprocessing when using fixed topic.